src/components/Popups/PopupReply.tsx

```diff
@@ -53,27 +53,18 @@ export default function PopupReply({
         handleEvent(event);
         onCancel(value);
     };
-    const handleFirstUpdate = (): void => {
-        const { current: textarea } = textareaRef;
-
-        if (textarea) {
-            textarea.focus();
-            textarea.selectionStart = value.length; // Force cursor to the end after focus
-            textarea.selectionEnd = value.length; // Force cursor to the end after focus
-        }
-    };
 
     return (
-        <PopupBase onKeyDown={handleKeyDown} options={{ onFirstUpdate: handleFirstUpdate }} {...rest}>
```

Comment:
Does re-focus on the text field still work properly without being tied to `onFirstUpdate`? I ran into issues with this previously.